db/migrate/058_name_unnamed_redactions.rb

```diff
@@ -0,0 +1,19 @@
+class NameUnnamedRedactions < ActiveRecord::Migration[5.0]
```

Comment:
The migration file should use a timestamp in the filename, see recent migrations that start with db/migrate/2018[...]